docs/migrations.rst

```diff
@@ -55,8 +55,6 @@ Phinx automatically creates a skeleton migration file with a single method:
              *    addIndex
              *    addForeignKey
              *
-             * Remember to call "create()" or "update()" and NOT "save()" when working
```

Comment:
Why has this been removed?

Reply:
I forgot to remove them in my previous PR, this is not required anymore